zeppelin/index.js

```diff
@@ -109,21 +114,33 @@ export default class LeafletMap extends Visualization {
 		map.invalidateSize(true)
 
 		var imageBounds = null
-		const markers = chartDataModel.rows.map(
+		const markers = chartDataModel.rows.flatMap(
 				row => {
 					const {image, boundary, info} = row;
+					var markers = [];
+
 					// throw new Error(image);
 					var jsts = require("jsts");
 					// Read WKT string from Sedona
 					var reader = new jsts.io.WKTReader();
-					var obj = reader.read(boundary)
+					var obj = reader.read(boundary);
 					// Collect the centroid point of the input geometry
 					var centroid = obj.getCentroid()
```

Comment:
Thank you for reviewing. Here is the point I want to discuss.If we don't plot centroid point, it will be hard for us to find the shape, so I keep it.

If we want to do better, we can choose whether to display the center point marker based on the zoom level, but implementing this will be more complicated.

Which solution do you prefer?

Reply:
Thanks. Your explanation makes sense to me. We can keep it this way. The marker can also be used to show tooltips, which is good.